commit f5ae83785b3dd9865d0448d1b6609fbbc4616055
Author: Michael Still <mikal@stillhq.com>
Date:   Tue Oct 7 10:31:12 2025 +1100

    Use local mirror of osbpo to work around reliability issues.

diff --git a/kolla/docker/base/Dockerfile.j2 b/kolla/docker/base/Dockerfile.j2
index 40fb6f9..1294c90 100644
--- a/kolla/docker/base/Dockerfile.j2
+++ b/kolla/docker/base/Dockerfile.j2
@@ -204,7 +204,7 @@ {% if base_distro == 'debian' %}
 
 RUN apt update \
  && apt install -y --no-install-recommends extrepo \
- && extrepo enable openstack_{{ openstack_release_codename | lower }} \
+ && extrepo enable openstack_{{ openstack_release_codename | lower }} --mirror http://mirror-osbpo.home.stillhq.com \
  && apt purge -y extrepo \
  && apt --purge autoremove -y \
  && apt clean
